test/unit/account_test.rb

```diff
@@ -21,6 +21,15 @@ class AccountTest < ActiveSupport::TestCase
     user.account.destroy
   end
 
+  test "create a new account" do
+    with_config invite_required: false do
```

Comment:
`with_config` says that, for this test, we assume the provider set `invite_required` to false in the configuration file.

This utility is specific to leap_web, have a look at its implementation.